solvebio/cli/data.py

```
@@ -645,6 +645,9 @@ def _download_recursive(
         file_obj.depth = depth
         # MD5 not retrieved by GlobalSearch so
         # separate API call is needed
+        if file_obj.class_name() == "Vault":
+            print("Do not processing Vaults, skipping {}".format(file_obj['vault']))
```

I don't think we need to print to warn users. We should just silently skip it.